STEP_Project/popup.js

```diff
@@ -27,9 +27,30 @@ function findPageType(givenurl)
     let settingspage = givenurl.match(/admin.google.com\/(u\/[0-9]\/)?ac\/appsettings\/[0-9]+\/.+/g);
     let onoffpage = givenurl.match(/admin.google.com\/(u\/[0-9]\/)?ac\/settings\/serviceonoff.*/g);
     pageType = PAGE_TYPES.OTHER;
+
     if (entitypage !== null)
     {
         pageType = PAGE_TYPES.ORG_LIST_PAGE;
+
+        let entityPageOU = givenurl.match(/admin.google.com\/(u\/[0-9]\/)?ac\/orgunits/g);
+        let entityPageUser = givenurl.match(/admin.google.com\/(u\/[0-9]\/)?ac\/users/g);
+        let entityPageGroup = givenurl.match(/admin.google.com\/(u\/[0-9]\/)?ac\/groups/g);
+
```

Comment:
We can reuse findPageType function of popup.js here. Consider moving that function to a shared js file, say common.js and move its tests there. Then. use that function here and where it is originally being used.

Reply:
This is part of the same function (findPageType). I've just increased the functionality.